### PR TITLE
Added french

### PR DESCRIPTION
**Pull request by magsout**
maybe @karlcow could take a look?

i'm not really sure about `refine`

### CODE REVIEW

**Review by wisniewskit — CHANGES_REQUESTED** [review]
This works for me, thanks (assuming karlcow is fine with the strings). But I'm afraid that I just added one more string for a general error to the list:
```
  "errorNoResultsAvailable": {
    "message": "No reduced results could be obtained."
  },
```

Sorry for the churn!

**Comment by magsout** [comment]
@wisniewskit done

File: _locales/fr/messages.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "message": "Inclure les @page"
+  },
+  "alsoIncludeScripts": {
+    "message": "Inclure les scripts"
+  },
+  "confirmLoseEdits": {
+    "message": "Perdre vos modification et restreindre à nouveau ?"
+  },
+  "couldNotAccessInspectedNode": {
+    "message": "Impossible d'acceder au noeud... cross-origin iframe?"
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Impossible d'accéder au nœud… cross-origin iframe?"
```

File: _locales/fr/messages.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{
+  "addonDescription": {
+    "message": "Extrait le code HTML et CSS pour générer des cas de test. "
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Extrait le code HTML et CSS pour générer un test."
```

File: _locales/fr/messages.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "addonName": {
+    "message": "Testcase Reducer"
+  },
+  "alsoIncludeAllMedias": {
+    "message": "Inclure les @media"
+  },
+  "alsoIncludeAncestors": {
+    "message": "Inclure les noeuds parents"
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Inclure les nœuds parent"
```

File: _locales/fr/messages.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "alsoIncludeCSSFonts": {
+    "message": "Inclure les polices CSS"
+  },
+  "alsoIncludeMetas": {
+    "message": "Inclure les meta tags important"
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Inclure les meta tags importants"
```

File: _locales/fr/messages.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "alsoIncludePageRules": {
+    "message": "Inclure les @page"
+  },
+  "alsoIncludeScripts": {
+    "message": "Inclure les scripts"
+  },
+  "confirmLoseEdits": {
+    "message": "Perdre vos modification et restreindre à nouveau ?"
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Perdre vos modifications initiales et réduire à nouveau ?"
```

File: _locales/fr/messages.json [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "couldNotAccessInspectedNode": {
+    "message": "Impossible d'acceder au noeud... cross-origin iframe?"
+  },
+  "errorNoResultsAvailable": {
+    "message": "Aucun résultat n'a pu être obtenu."
+  },
+  "errorUnexpectedCSSRule": {
+    "message": "Testcase Reducer a trouvé une règle CSS inattendue :"
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Testcase Reducer a trouvé une règle CSS inattendue : "
```

File: _locales/fr/messages.json [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  "showSameViewportSize": {
+    "message": "Afficher l'iframe dans sa taille original "
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Afficher l'iframe dans sa taille originale"
```

File: _locales/fr/messages.json [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+    "message": "Envoyer sur JSBin"
+  },
+  "openInJSFiddle": {
+    "message": "Envoyer sur JSFiddle"
+  },
+  "openInNewTab": {
+    "message": "Ouvrir dans un nouvel onglet"
+  },
+  "reduce": {
+    "message": "Restreindre"
```
Comment (karlcow) [inline _locales/fr/messages.json]:
```suggestion
    "message": "Réduire"
```

**Review by karlcow — CHANGES_REQUESTED** [review]
@magsout @wisniewskit Some suggestions

**Comment by magsout** [comment]
@karlcow oups sorry the misreading.
fixed and committed, thanks for your review.

r @wisniewskit ?

**Comment by wisniewskit** [comment]
Works for me, thanks! Just FYI, I'll try to ping you whenever I'm going to roll another release, and there are new strings to localize.

**Pull request merged**